### ISSUE TITLE
resolve_type missing in Relay documentation

### ISSUE THREAD

**Issue by shaimo** [rmosolgo/graphql-ruby#586]
I was pulling my hair for quite long until I realized that was the issue...
If you don't implement it you will get:
```
NotImplementedError: schema contains Interfaces or Unions, so you must define a `resolve_type -> (obj, ctx) { ... }` function
```

**Comment by rmosolgo** [comment]
Oh, I'm sorry to hear that! Did the error message help you find the next step? 

Perhaps it should include a link to the [appropriate documentation](https://rmosolgo.github.io/graphql-ruby/schema/configuration_options#execution-functions)

**Comment by shaimo** [comment]
@rmosolgo not exactly. The error didn't indicate which interface or union caused the issue so I had to go and comment out most of my schema to try and isolate it. Eventually I figured it must be the GraphQL::Relay::Node.interface interface. Looking into it I couldn't figure what I needed to implement other than including a global_id_field, which I already did. Eventually I found some other issue that someone submitted that pointed to an example that did implement the resolve_type function so I figured I was missing it... Thanks
